src/hub/atlas.js

```diff
@@ -421,6 +421,73 @@ class AtlasClient {
         return users;
     }
 
+    /**
+     * Update an existing users data
+     *
+     * @param {Object} data - A user data object as specified for the v1/user/ endpoint PATCH method
+     * 
+     * @returns {Boolean} indicates whether the patch was successful or not
+     */
+    async patchUser(data) {
+        const op = { method: "PATCH", body: { ...data } };
+        try{
+            await this.fetch("/v1/user/" + data.id, op);
+            return true;
```

Comment:
Don't we need to check the response from the server?  E.g. `if (!resp.ok) { throw new Error("BADDY");}`

Reply:
this.fetch might already do this resp.ok check actually.